test/fixtures/output/api-key/47.golden

```diff
@@ -1,4 +1,4 @@
-Error: error getting API key: resource not found: unknown API key UNKNOWN
+Error: error getting API key UNKNOWN: resource not found: unknown API key UNKNOWN
```

Comment:
It would be great to put quotes around all resources!
```suggestion
Error: error getting API key "UNKNOWN": resource not found: unknown API key "UNKNOWN"
```

Reply:
Agreed. But I think I'll leave this one alone, because half of this is the backend error that doesn't have quotation marks. I checked that the real backend returns the same thing.

I think we can probably toss the backend error, but I'm not sure if there are other (meaningful) messages that it might return aside from `unknown API key %s`